app/src/test/java/org/oppia/android/app/activity/route/ActivityRouterModuleTest.kt

```diff
@@ -102,17 +109,36 @@ class ActivityRouterModuleTest {
   }
 
   @Test
-  fun testInjectActivityRouterModule_isInstanceOfActivityRouter() {
-    val activityRouter = retrieveActivityRouter()
-    assertThat(activityRouter).isInstanceOf(ActivityRouter::class.java)
+  fun testInjectActivityRouterModule_routeToRecentlyPlayedActivity() {
```

Comment:
We care less about routing as presence. Suggest instead:

```suggestion
  fun testInjectDestinationRoutes_hasRouteForRecentlyPlayedActivity() {
```

Ditto below (the other name should match similarly).

Reply:
Updated